gems/pending/VolumeManager/MiqVolumeManager.rb

```diff
@@ -124,7 +125,7 @@ def parseLvmMetadata(pvHdrs)
       if pvh.lvm_type == "LVM2"
         $log.debug "MiqVolumeManager.parseLvmMetadata: parsing LVM2 metadata"
         pvh.mdList.each do |md|
-          # Lvm2MdParser.dumpMetadata(md)
+          Lvm2MdParser.dumpMetadata(md)
```

Comment:
Do we want to unconditionally dump the metadata listing to the log? If we keep this call, should we change `dumpMetadata` to log on `debug` instead of `error`? Then this line could be changed to:
```
Lvm2MdParser.dumpMetadata(md) if $log.debug?
```